ftp/1.5/modules/ROOT/pages/ftp-dynamic-connection.adoc

```diff
@@ -0,0 +1,159 @@
+= Connect to FTP Servers Dynamically Example - Mule 4
+
+Many integrations require connecting to different servers depending on a certain condition.
+
+Examples of this include:
+
+* Connecting to different invoice storage servers depending on the branch that emits an invoice.
+* Connecting to different servers depending on an integration subject, such as in a multitenant use case.
+
+To accommodate these use cases, the `config` element supports expressions, which allows for connection parameters to evaluate these conditions and connect to the correct server.
+
+The following example shows a dynamic multicast application that:
+
+* Defines a config for the connector in which the `host`, `username`, and `password` are expressions.
+* Describes a flow, triggered by an *HTTP Listener*, in which content is posted through HTTP.
+* Uses the File Connector *Read* operation to load a recipients CSV file that contains a set of FTP destinations, with columns such as `host`, `user`, and `port`.
+* Uses a *For Each* component and an FTP *Write* operation to write the contents into each of the FTP destinations.
+** On each *For Each* iteration, the expressions set in the FTP global configuration element resolve to a different value and establish different connections to each of the servers.
+
+To test this example, create the Mule application, and run and test the application with curl commands:
+
+image:ftp_dynamic_connect_flow.png:[FTP Servers Dynamic Connection Flow in Studio Canvas]
+
+== Create the Mule Application
+
+To create the Mule flow:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *HTTP Listener* to the Studio canvas.
+. On the *HTTP Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Set the *Path* field to `/multitenant`
```

Comment:
```suggestion
. Set *Path* to `/multitenant`.
```

Reply:
Per Style guide now consider referring to UI elements by label rather than element type:

Before
. Select the Upload all my files now checkbox and click the Next button.

After
. Select Upload all my files now and click Next.